cli.js

```diff
@@ -115,15 +120,30 @@ if (shouldWatch) {
       track,
     })
   } else {
-    const { code } = morph(readFileSync(input, 'utf-8'), {
-      as,
-      compile,
-      file: { raw: input, relative: input },
-      name: pathToName(input),
-      pretty,
-      track,
-    })
-
-    console.log(code)
+    if (input.includes('.svg')) {
+      return new Promise(async (resolve, reject) => {
+        const inlined = await morphInlineSvg(input)
+
+        resolve(
+          onMorphWriteSvg({
+            as,
+            code: inlined,
+            isInlineSvg: true,
+            file: path.resolve(input, '..', `${pathToName(input)}.view`),
+          })
+        )
+      })
```

Comment:
Is there any reason why you wrapped the whole thing in a promise? it seems to me that you can get away with doing something like:
```
morphInlineSvg(input).then(code => writeFile(path.resolve(input, '..', `${pathToName(input)}.view`), code)
```

Also, when we morph a view in the next bit of logic below, we're outputting the code to the console, so you can do like in unix tools and pipe it to something else like a file or copy it from the terminal to something else. Ideally we'd keep the two of them consistent. What're your thoughts on this?

Thanks for bringing this back 😄